packages/patternfly-4/react-core/src/components/Tabs/TabContent.js

```diff
@@ -1,36 +1,53 @@
-import React from 'react';
+import * as React from 'react';
 import PropTypes from 'prop-types';
 import { css } from '@patternfly/react-styles';
 
 const propTypes = {
-  /** the index of the active tab */
+  /** the index of the active tab (not needed if used outside Tabs component) */
   activeKey: PropTypes.number,
-  /** content rendered inside the tab content area. */
+  /** content rendered inside the tab content area if used inside Tabs component */
   child: PropTypes.node,
+  /** content rendered inside the tab content area if used outside Tabs component */
+  children: PropTypes.any,
```

Comment:
While I don't think we need separate components for Tabs with div/nav, this one looks like we could split off, maybe a TabOutsideContent